.yarn/patches/@metamask-queued-request-controller-npm-0.10.0-a228bc1b3b.patch

```diff
@@ -0,0 +1,1508 @@
+diff --git a/dist/QueuedRequestController.js b/dist/QueuedRequestController.js
```

Comment:
This patch looks like it contains a bunch more than it needs to? Seems to include way more than just the lines that are [changed in the core side PR](https://github.com/MetaMask/core/pull/4423)?

Reply:
the outputted chunk names are entirely different depending on the contents of the chunk, and the diff is not able to reconcile this as a file rename + line changes, instead calling it a file delete and a file create

Reply:
Linking [this thread](https://consensys.slack.com/archives/CTQAGKY5V/p1718123750012709) on why this kindof (full override) patch for posterity